torch/nn/modules/conv.py

```diff
@@ -579,8 +579,8 @@ def __init__(self, in_channels, out_channels, kernel_size, stride,
             padding, dilation, transposed, output_padding,
             groups, bias, padding_mode)
 
-    def _output_padding(self, input, output_size, stride, padding, kernel_size):
-        # type: (Tensor, Optional[List[int]], List[int], List[int], List[int]) -> List[int]
+    def _output_padding(self, input: Tensor, output_size: Optional[List[int]], stride: List[int],
+                        padding: List[int], kernel_size: List[int]) -> List[int]:
```

Comment:
Kernel_size, stride, and padding passed here are actually Tuple[int, ...], but Torchscript doesn't seem to support arbitrary length tuples. Based on #14775, seems like it's necessary to type these as List[int] for torchscript to convert arguments to lists.
This requires changing the attribute type to Any above to avoid type mismatch errors.

Reply:
One can use `cast` to convert `List` to `Tuple`